must-gather/collection-scripts/gather_nro

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# get crds
+for crd_name in $(oc get crds -o=custom-columns=NAME:.metadata.name --no-headers | grep -E "^*\.nodetopology\.openshift\.io$")
+do
+    oc adm inspect --dest-dir must-gather crd/${crd_name}
+done
+
+# Get NRO namespace and gather all the data there
+. namespace
```

Comment:
Maybe I'm missing something here, but this is a bash script.
Is there such a command ```. namespace```?

Reply:
Yep, executing this: https://github.com/openshift-kni/performance-addon-operators/blob/master/must-gather/collection-scripts/namespace

Reply:
the command is actually `.` (dot), aliased by `source`, which means: "read and execute in the current shell context the given file"

Reply:
But the function name has just changed

Reply:
Maybe not the best filename to avoid confusions.